src/modules/configuration/index.ts

```diff
@@ -406,6 +407,15 @@ export const configValues = () => ({
           firstDepositId: 1,
         },
       ],
+      [ChainIds.alephZero]: [
+        {
+          address: "0x09aea4b2242abC8bb4BB78D537A67a245A7bEC64",
+          startBlockNumber: 4112376,
+          abi: JSON.stringify(SpokePoolV3Abi),
+          acrossVersion: AcrossContractsVersion.V3,
+          firstDepositId: 0,
```

Comment:
Does this fit with the deploy we made?

Reply:
Is `firstDepositId` 0 or 1?

Reply:
> Does this fit with the deploy we made?
Yes, do you see any differences? @james-a-morris

Reply:
> Is firstDepositId 0 or 1?

We set it to 0, [here](https://github.com/across-protocol/contracts/pull/704/files#diff-96d466a4a26299aa39f73fd3c54562c8630ac95ed678f171a0cb619e8717bb91R9-R15). The first item of `initArgs` is the `initialDepositId`